applications/GeoMechanicsApplication/tests/test_pressure_line_element/test_oblique_line_element2D2N/MaterialParameters.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
If this material file is places one directory up, the other 2D computations can reuse it.